CMakeLists.txt

```diff
@@ -488,6 +490,18 @@ endif()
 #       - Decoded cert cache
 #       - SEP
 
+set(WOLFSSL_CERTGEN_HELP_STRING "Enable cert generation (default: disabled)")
```

Comment:
Likewise for the TODOs here and elsewhere.

Reply:
Done